### DISCUSSION TITLE
Launch vmware-view

### DISCUSSION THREAD

**Discussion by solune** [ltsp/ltsp#186] [Q&A]
I'm using LTSP to transform PC in vmware thin client. 
With old LTSP, I was using this method to launch vmware-view at session start: https://aggroworld.org/2016/01/10/vmware-horizon-view-linux-ubuntu-ltsp-thinclient/

With the new version, How can I do ?
Thanks

**Comment by alkisg** [comment]
See https://github.com/ltsp/ltsp/issues/39#issuecomment-539150708 for a method to create an xfreerdp.desktop session, and a user that would autologin to it.

It's the same for any kind of kiosk-like screen script; you create a session.desktop file that runs whatever you need, and you autologin to it.
